lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -171,8 +171,9 @@ class FlutterViewEmbedder {
     _glassPaneShadow = glassPaneElementHostNode;
 
     // Don't allow the scene to receive pointer events.
-    _sceneHostElement = domDocument.createElement('flt-scene-host')
-      ..style.pointerEvents = 'none';
+    _sceneHostElement = domDocument.createElement('flt-scene-host');
+      // DO_NOT_SUBMIT: revert this before submitting.
+      // ..style.pointerEvents = 'none';
```

Comment:
fwiw, one trick for this kind of thing is to use `// TODO: revert before submitting` because then the todo style lint will make the analyzer shard red, preventing you from landing.